sdks/python/apache_beam/runners/worker/sdk_worker_main.py

```diff
@@ -252,7 +252,8 @@ def _load_main_session(semi_persistent_directory):
     else:
       _LOGGER.warning(
           'No session file found: %s. Functions defined in __main__ '
-          '(interactive session) may fail.',
+          '(interactive session) may fail. Ignore this warning'
```

Comment:
We should be able to skip the `_load_main_session` call or not print the warning, so that the users don't have to worry about understanding what pickle_library is when we switch the default to cloudpickle.

Reply:
Added an If condition to check whether the pickle_library is cloud_pickle or not.